_i18n/en/documentation/bugs-first-aid/refresh-download-errors.md

```diff
@@ -0,0 +1,12 @@
+Do you get a notification saying that downloads completed with errors? Then AntennaPod could not retrieve the feed with episode information or an audio files from the website of the podcast hoster.
```

Comment:
```suggestion
Do you get a notification saying that downloads completed with errors? Then AntennaPod could not retrieve the feed with episode information or an audio file from the website of the podcast hoster.
```

The word "feed" is very technical. I would try to avoid it, or at least add "RSS". (AntennaPod supports other feed formats as well but I think RSS is what one sometimes sees on websites)